lib/money/money.rb

```diff
@@ -34,6 +34,96 @@ def *(other)
     end
   end
 
+  class SplitList
```

Comment:
I'd suggest moving this concern to a new `money/splitter.rb` file, similar to https://github.com/Shopify/money/blob/main/lib/money/allocator.rb. 
This class could have two public methods `split` and `calculate_split`.

Reply:
Done.